api/__tests__/services/vimeo.test.js

```diff
@@ -0,0 +1,44 @@
+const { faker } = require('@faker-js/faker');
+const axios = require('axios');
+const vimeo = require('../../services/vimeo');
+
+
+describe('Test the Vimeo service', () => {
+
+  test('getVideoFileFromVimeo', async () => {
+    // Set up fake test data
+    const fakeVimeoId =   faker.datatype.number({ min: 100000000, max: 999999999 });
+    const fakeVideoWebPage = `https://vimeo.com/${fakeVimeoId}`;
+    const fakeApiVimeoCall = `https://player.vimeo.com/video/${fakeVimeoId}/config`;
+    const fakeVimeoParameter = {Accept: 'application/json'};
```

Comment:
Optional tiny thing, but could rename this something like `fakeApiVimeoOptions` or `fakeAxiosConfig` (whatever you think is clearest) here and where it appears below, just so it's even clearer (as parameter is a more general term).